tests/drivers/counter/counter_basic_api/boards/nrf54l15bsim_nrf54l15_cpuapp.overlay

```diff
@@ -1 +1 @@
-#include "nrf54l15pdk_nrf54l15_cpuapp.overlay"
+#include "nrf54l15dk_nrf54l15_cpuapp.overlay"
```

Comment:
Together with this correction, you should also rename that included file itself (instead of removing it in the next commit).

Reply:
@anangl done